src/audio/pcm_converter/pcm_converter_hifi3.c

```diff
@@ -0,0 +1,643 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2019 Intel Corporation. All rights reserved.
+//
+// Author: Tomasz Lauda <tomasz.lauda@linux.intel.com>
+
+/**
+ * \file audio/pcm_converter/pcm_converter_hifi3.c
+ * \brief PCM converter HiFi3 processing implementation
+ * \authors Tomasz Lauda <tomasz.lauda@linux.intel.com>
+ */
+
+#include <sof/audio/pcm_converter.h>
+
+#ifdef CONFIG_PCM_CONVERTER_HIFI3
+
```

Comment:
Why do you need this #ifdef here? Just compile out the entire file if CONFIG_HAVE_PCM_CONV_HIFI3 is not set.

Reply:
@dbaluta It's coming from header, not config. Fixed name.